src/asciidoc/docs/develop/app-dev-process.adoc

```diff
@@ -0,0 +1,161 @@
+= App development process
+:page-slug: app-dev-process
+:page-description: Overview of how to develop a Koji app, from forking an existing app, to making your app customizable and enabling platform features, to deploying, testing, and distributing it.
+:includespath: ../_includes
+
+The Koji platform is designed for developers like you to build and distribute customizable web apps for content creators.
+On Koji, creators can discover your apps, and then add them to their Link in Bio profiles and share them on social media.
+As more creators use your Koji apps, you can make more revenue, reach more users, and build your business.
+
+The following steps describe the general process for developing a customizable app on Koji and distributing it to content creators.
+
+== [.callout1]#Choose a starting point#
+
+A good starting point will provide a foundation that lets you start coding your app's unique features more quickly.
+For example, you might start from a https://withkoji.com/create/for-developers[developer scaffold] that implements your favorite JavaScript framework, or from an app that already uses some desired functionality, such as a backend database.
+
+. To create a new project, fork an existing app that allows developers to fork it.
+
+.. In the App Store, find your desired starting app and click btn:[View].
++
+TIP: To explore the source code for an app before forking it, click menu:...[Advanced Details > Source Code].
+If this option is not available, the app does not allow developers to fork it.
+
+.. In the app's detail page, click menu:...[Fork].
++
+A copy of the project is created in your Koji account.
+
+. Decide how you want to edit your code.
+* *Online* – The Koji code editor is a browser-based integrated development environment (IDE) with built-in development, administration, and testing tools for Koji apps.
++
+Learn more: <<editor#>>
+* *Locally* – The Koji platform supports working locally in your preferred suite of development tools.
++
+Learn more: <<local-development#>>
+
+== [.callout2]#Develop your app#
+
+At their core, Koji apps are full-stack, modern JavaScript applications.
+They are designed to run in webviews, so that they are instantly available on all social platforms and every device.
```

Comment:
Definitely a term to add to your list :) I had found a similar resource and used the capitalization you suggest for a previous document. At the time, Sean thought the generic would be more readable, so I changed it ([link to the discussion, for reference](https://github.com/madewithkoji/docs/pull/174#pullrequestreview-667366107)). I think either form would be fine, as long as we standardize. So, if Sean agrees, I'll change it in both places. Otherwise, I'll keep it and you can discuss in future style standards meetings.

Reply:
One suggestion might be to avoid `WebView`/`webview` (which feels like it refers more to the implementation inside the app - e.g., `WebView` is a class name on Android) and use something like `in-app browser`/`in-app browsers`. Seems like the phrase is common: https://www.google.com/search?q=in-app+browser

Reply:
Updated both pages this way. Thanks!